Resources/Prototypes/CharacterItemGroups/Jobs/Medical/uncategorized.yml

```
@@ -135,3 +138,9 @@
       id: LoadoutMedicalUniformScrubsCybersun
     - type: loadout
       id: LoadoutMedicalUniformScrubsWhite
+    # WD EDIT START
+    - type: loadout
+      id: LoadoutClothingUniformJumpsuitMedicalBlue
+    - type: loadout
+      id: LoadoutClothingUniformJumpsuitMedicalGreen
+    # WD EDIT END
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)